nightwatch/features/step_definitions/backendPosts.js

```diff
@@ -49,15 +49,28 @@ module.exports = function () {
   })
 
   this.When(/^I submit the following prebookings:$/, function (table) {
+    var validTimestamp = moment().set({hour: 7, minute: 0, second: 0}).format()
```

Comment:
please continue to keep the parameter of time in the test rather than specify past/future/valid in the code.
the first question someone reading it will have is "what is considered past/etc" we can use the test to describe that and then parse a human readable string into a timestamp to send